src/main/java/ser/mil/rideapp/domain/repository/RideRepository.java

```diff
@@ -19,4 +20,7 @@ public interface RideRepository {
     List<Ride> pendingRides(Provider provider);
 
     List<Driver> availableDrivers(Provider provider);
+
+    Driver findDriverById(String id);
```

Comment:
Jeśli repo ma metode"findCosTam" to powinno zwracać Optional<>, jesli "getCosTam" to zwraca obiekt/null/exception